templates/tfengine/recipes/project.hcl

```diff
@@ -87,7 +87,7 @@ schema = {
                 description = "The API to apply the roles to. Example: healthcare.googleapis.com"
               }
               roles = {
-                description = "Roles to granted to the API Service Agent."
+                description = "The API whose default Service Agent will be force-created and granted the role. Example: healthcare.googleapis.com."
```

Comment:
Should be updated on the `api` field.

Reply:
You're right. I'm so sorry, didn't catch it D: